src/sugar3/graphics/radiopalette.py

```diff
@@ -22,6 +22,10 @@
 
 
 class RadioMenuButton(ToolButton):
+    '''
+    This is a button used to open a group of widgets in a RadioPalette(menu)
+    It is not clickable
```

Comment:
What are the kwargs that I should pass to it?  Maybe give a 1 line example of construction?

Also, what does it do?  You say it opens a group of buttons (what does this mean?) but is not clickable (my mind is very confused).